locations/searchable_points/postcodes/README.md

```diff
@@ -0,0 +1,18 @@
+
+## Postal code data
+
+Postal code data can be very useful for driving store locator query
+interfaces which require discrete area parameters. These could be
+postal codes or lat/lon.
+
+There are various open data releases of postal code data. Available
+here:
+
+**uszips.csv** "BASIC" download from [simplemaps](https://simplemaps.com/data/us-zips)
```

Comment:
This file is licensed CC-BY 4.0. We would need to provide attribution somewhere, maybe in this code? I haven't had a chance to dig into what would be required.

Reply:
I had noted that an added the README.md in the same directory. I will add more links in the appropriate code fragment.

Reply:
Added more. I believe we were good before .... but even better now.

Reply:
Marking this as unresolved after having a quick read up on GitHub etiquette ..... new to GitHub you see!